src/css/bloblang-playground.css

```
@@ -1067,35 +1178,35 @@ html[data-theme=dark] .bloblang-playground .notification-info {
 html[data-theme=dark] .bloblang-playground .validation-success {
   color: var(--color-primitives-global-green-green-400);
   background: rgba(56, 161, 105, 0.15);
 }
 
+/* Error state - allow full message to show */
 .bloblang-playground .validation-error {
   color: var(--color-primitives-global-red-red-700);
   background: var(--color-primitives-global-red-red-50);
-  max-width: 100%;
+  white-space: normal;
   word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)